### PR TITLE
Adjust menu items

### LINKED ISSUE #492: :iphone: Adjust menu items
###How it should look
Main-Home

###How it looks like
image

*The difference:*
image

###Figma design
https://www.figma.com/file/pxj9Hhk5AdOxXWJP0dq4gT/Library-App-Spec?node-id=4932%3A14059&t=QwPMxPupTSLxT5nT-0

### PR DESCRIPTION
**Pull request by Chichmarin17**
Closes #492  
image

### CODE REVIEW

File: packages/frontapp/src/app/components/Navbar/index.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const NavbarBottomSubsectionWrapper = styled.div`
   position: fixed;
   bottom: 0;
 
-  @media (max-height: 747px) {
+  @media (max-height: 784px) {
```
Comment (sabrina34352) [inline packages/frontapp/src/app/components/Navbar/index.tsx:36]: we have `dimensions.ts` file, where we store all the sizes of components and containers, i would recommend moving max-height to dimensions.ts and use it as import

**Review by sabrina34352 — APPROVED** [review]

File: packages/frontapp/src/app/components/Header/index.tsx [hunk 2]
```diff
@@ -37,10 +37,11 @@ const WrapperForCenter = styled.div`
 const WrapperHeader = styled.header`
   max-width: 16.5rem;
   width: 100%;
-  margin-bottom: 2rem;
+  margin-bottom: 3.5rem;
   @media (max-width: ${dimensions.tablet_width}) {
     max-width: none;
     padding: 0 20px;
+    margin-bottom: 2.5rem;
```
Comment (mabrur-h) [inline packages/frontapp/src/app/components/Header/index.tsx:44]: I would recommend to use variables from dimensions file.

**Review by mabrur-h — APPROVED** [review]

File: packages/frontapp/src/app/components/Navbar/index.tsx (hunk 1 above)
Comment (Chichmarin17) [reply packages/frontapp/src/app/components/Navbar/index.tsx:36]: Can't find any usage for that dimension, but I will consider that

File: packages/frontapp/src/app/components/Header/index.tsx (hunk 2 above)
Comment (Chichmarin17) [reply packages/frontapp/src/app/components/Header/index.tsx:44]: Make sense

**Pull request merged**